stdlib/camlinternalFormatBasics.mli

```diff
@@ -205,6 +210,9 @@ and ('a, 'b, 'c, 'd, 'e, 'f) fmt =
 | Bool :                                                   (* %[bB] *)
     ('x, bool -> 'a) padding * ('a, 'b, 'c, 'd, 'e, 'f) fmt ->
       ('x, 'b, 'c, 'd, 'e, 'f) fmt
+| Binary :                                                 (* %Y *)
```

Comment:
I don't understand why you use `Binary` and that sounds wrong to me, why not use `Bytes` instead?

Reply:
no probs, I renamed to Bytes_ty and Bytes